pkg/domain/attributor.go

```diff
@@ -99,46 +99,52 @@ func (n *CloudAssetDetails) UnmarshalJSON(data []byte) error {
 // returns either a 404 Not Found response, or a 200 OK response with no results
 type AssetNotFoundError struct {
 	Inner          error
-	AssetID        string
-	ScanTimestamp  string
 	AssetInventory string
 }
 
 func (err AssetNotFoundError) Error() string {
 	return fmt.Sprintf(
-		"Result not found for asset with ID %s as of scan time %s in asset inventory %s: %v",
-		err.AssetID, err.ScanTimestamp, err.AssetInventory, err.Inner)
+		"Result not found for asset asset inventory %s: %v",
```

Comment:
There's a grammatical error here I think.  What is the error message supposed to say?

Reply:
yea... this makes 0 sense. fixed!